patterns/pattern-2/template.yaml

```diff
@@ -150,6 +150,11 @@ Resources:
             required:
               - features
             properties:
+              tuning:
```

Comment:
Confused by 'tuning'  - seems disconnected from enable true | false which is supposed to be the focus of this PR.  Please remove all this (looks like hallucination or scope creep) and ensure PR is laser focused on just the classification/extraction enable/disable toggle.

Reply:
Not resolved - reopening.